autorest/date/date_test.go

```diff
@@ -3,22 +3,30 @@ package date
 import (
 	"encoding/json"
 	"fmt"
-	// "github.com/Azure/go-autorest/autorest"
-	// "github.com/Azure/go-autorest/autorest/mocks"
 	"reflect"
 	"testing"
 	"time"
 )
 
 func ExampleParseDate() {
-	d, _ := ParseDate("2001-02-03")
+	d, err := ParseDate("2001-02-03")
+	if err != nil {
+		fmt.Println(err.Error())
```

Comment:
you don't need to call `Error()` on errors. https://play.golang.org/p/Y2J7YdIFZU